domainlab/algos/trainers/train_fishr.py

```diff
@@ -161,10 +161,6 @@ def cal_dict_variance_grads(self, tensor_x, vec_y):
                 inputs=list(self.model.parameters()), retain_graph=True, create_graph=True
             )
 
-        for name, param in self.model.named_parameters():
```

Comment:
we don't need these lines anymore?

Reply:
This is just printing and does not even use a logger, so it appears to be debug code? Can add it within a logger if you want.

Reply:
you have another function already implemented the print hyperparameters?

Reply:
you are right, that line seems to have no effect.